### ISSUE TITLE
Entry Field Placeholders

### ISSUE THREAD

**Issue by Solifice** [Kunzisoft/KeePassDX#1503]
Recently I wanted to link my few entries with each other, then I come to know that Keepass supports Field References. Later I also learned about the Placeholders in Keepass as well. It solved my problem. Both Field reference and Entry Field Placeholders were working fine on my KeepassXC on PC. Later day when I checked on my smartphone on KeepassDX. I saw that Entry Field Placeholders was not working as expected. My entry was looking like this :- {USERNAME}@gmail.com. I tried this thing on few entries again. And same happened with all them. Field References were working fine. I think this feature was not added in KeepassDX, therefore raise a feature request, if its a bug please move this to bug report.

![image](https://user-images.githubusercontent.com/59728110/217855010-a6778735-a285-4b1d-bbd4-871c5cf3fd1f.png)

**Comment by J-Jamet** [comment]
Indeed, the field references are already managed but not the placeholders.

**Comment by ishaankaushal-nagarro** [comment]
@J-Jamet I also noticed that {S:<custom_string_name>} also doesn't works, so you can add this also. BTW, really an excellent work on this. Other Keepass managers for android are dead and hardly receive updates. I hope so, I will get this update soon ;). Would like to donate this project once I get placed in a company 💖.

**Comment by Solifice** [comment]
@J-Jamet I also noticed that {S:<custom_string_name>} also doesn't works, so you can add this also. BTW, really an excellent work on this. Other Keepass managers for android are dead and hardly receive updates. I hope so, I will get this update soon ;). Would like to donate this project once I get placed in a company with a good package 💖.
